dev/io.openliberty.faces.4.0.thirdparty/bnd.bnd

```diff
@@ -39,7 +39,5 @@ publish.wlp.jar.suffix: dev/api/third-party
 
 -fixupmessages.missingexport: "Used bundle version * for exported package";is:=ignore
 
--buildpath: org.apache.myfaces.core:myfaces-impl;version=${Implementation-Version},\
-            org.apache.myfaces.core:myfaces-api;version=${Implementation-Version}, \
-            io.openliberty.jakarta.cdi.4.0;version=latest, \
-            io.openliberty.jakarta.expressionLanguage.5.0;version=latest
+-buildpath: io.openliberty.org.apache.myfaces.4.0;version=project, \
+    org.apache.myfaces.core:myfaces-impl;version=${Implementation-Version}
```

Comment:
Reverted the bnd so that the overlays would be picked up from the 4.0 implementation project.